src/components/tags/tags.js

```diff
@@ -0,0 +1,107 @@
+import template from "./tags.html";
+import tagItemTemplate from "./tag-item.html";
+import { ButtonMoreComponent } from "../button-more/button-more";
+import { TagListService } from "../../services/tag-service";
+import { AddTagDialogComponent } from "../add-tag-dialog/add-tag-dialog";
+
+export class TagsComponent {
+  constructor(mountPoint) {
+    this.mountPoint = mountPoint;
+  }
+
+  querySelectors() {
+    this.tagsRoot = this.mountPoint.querySelector(".tags");
+    this.moreBtnMountPoints = this.mountPoint.querySelectorAll(
```

Comment:
you don't need this, because you firstly mount empty component (no buttons to search), then you load tags from storage and get your buttons